### ISSUE TITLE
Add UI tests to CI

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#2007]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation
The lack of UI tests in the CI process makes it difficult to identify bugs before release. That leads to severe bugs likely to make it into release, e.g.:
- https://github.com/parse-community/parse-dashboard/issues/1991
- https://github.com/parse-community/parse-dashboard/issues/1994
- https://github.com/parse-community/parse-dashboard/issues/2006
- https://github.com/parse-community/parse-dashboard/issues/2194

The current manual testing before merging a PR is not reliable nor scalable.

### Feature / Enhancement Description
Add UI tests to CI, starting with just a few basic tests as guidance on how to write these tests, that can be extended over time.

### Example Use Case
n/a

### Alternatives / Workarounds
Manual testing which is not reliable or scalable.

### 3rd Party References
n/a

**Comment by paulovitin** [comment]
@mtrezza, Are you talking about Cypress or Integration with [RTL](https://testing-library.com/docs/react-testing-library/intro/) ecosystem?

**Comment by mtrezza** [comment]
See https://github.com/parse-community/parse-dashboard/pull/2030, something like that.

**Comment by mehulmathur16** [comment]
Hi @mtrezza, is this issue still open ?

**Comment by mtrezza** [comment]
Yep, it is, but there has been at least 1 PR: https://github.com/parse-community/parse-dashboard/pull/2030, not sure what the state of that is.

**Comment by mehulmathur16** [comment]
Okay @mtrezza. I'll give this a try.

**Comment by mtrezza** [comment]
@mehulmathur16 fantastic, please take a look at the discussion in #2030, it contains some feedback regarding how to implement this.
